### ISSUE TITLE
"telephone" whisper component behave incorrect for tooltip and value

### ISSUE THREAD

**Issue by hans-hsieh-asus** [open-olive/loop-development-kit#138]
When using `Telephone` whisper component, it looks to have some problems. With the code below
```javascript
await whisper.create({
      label: 'Telephone',
      components: [
        {
          type: whisperService.WhisperComponentType.Markdown,
          body: 'This is a "telephone" component without box.',
        },
        {
          type: whisperService.WhisperComponentType.Telephone,
          label: 'label',
          onChange: (value) => console.log(`Telephone is changed: ${value}`),
          tooltip: 'tooltip',
          value: '09123456789',
        },
      ],
      onClose: () => {},
    });
```
, I see
![Untitled](https://user-images.githubusercontent.com/68371064/116856137-2d648400-ac2d-11eb-8492-ed88b0ea2cbb.png)
And there's no tooltip.

LDK version: 3.0.0-beta.1
OliveHelps version: 0.16.2

**Comment by cshields-oliveai** [comment]
Thanks for including an image with this one too!

We have opened a new LDK ticket to investigate and resolve: SIDE-2189.
As with other your other reported issues, we will provide updates in this issue as we work to get it resolved, and will provide a pull request link if any work touches the LDK for your reference.

Expect to hear from us soon.

**Comment by BrettMugglinOlive** [comment]
This should be fixed in the next release of Olive Helps!

**Comment by CandiceCLB** [comment]
This issue has been resolved with the latest Olive Helps release V 0.17.3. If you find you are still having this issues, please feel free to re-open the ticket.
